LDRH_choice.py

```diff
@@ -456,7 +456,10 @@ def run_staircase(task, operation=None):
         if operation: output_score = [output['score'] if i==order_dict[operation] else None for i in range(1,5)]
         else: output_score = output['score']
 
-        handler.addData(output['score'])
+        if use_posterior_matching: 
+            if operation: handler.addData(operation,output['score'])
```

Comment:
Right, so we could pass this information at initialization, rather than having to do it every time we pass in data.